### PR TITLE
updated text file

### PR DESCRIPTION
**Pull request by tramstheman**
I updated the text file to reflect all the changes made.

### CODE REVIEW

**Comment by tramstheman** [comment]
can you comment on the line you are referring to?

File: Assignment_turn_in_module3.txt [hunk 1]
```diff
@@ -156,14 +151,18 @@ public class Main
       {
          System.out.printf("The %s was pulled from the deck.\n", myDeck2.dealCard());
       }
+
+      //Tests boundaries of numPacks.
+      myDeck2.init(10);
```
Comment (GEvanscsumb) [inline Assignment_turn_in_module3.txt]: this check and the one below are not in the sample output

File: Assignment_turn_in_module3.txt [hunk 2]
```diff
@@ -240,148 +240,161 @@ public class Main
...
    }
-   
+
    /**
     * Public Copy Constructor
-    * @param originalCard values to be copied into new separate com.northvine.engineering.cards.Card object
+    *
+    * @param originalCard
+    *           values to be copied into new separate
+    *           com.northvine.engineering.cards.Card object
```
Comment (GEvanscsumb) [inline Assignment_turn_in_module3.txt]: this is the package path I was referring to

File: Assignment_turn_in_module3.txt [hunk 3]
```diff
@@ -156,14 +151,18 @@ public class Main
       {
          System.out.printf("The %s was pulled from the deck.\n", myDeck2.dealCard());
       }
+
+      //Tests boundaries of numPacks.
+      myDeck2.init(10);
+      myDeck2.init(0);
```
Comment (GEvanscsumb) [inline Assignment_turn_in_module3.txt]: not in sample output

File: Assignment_turn_in_module3.txt (hunk 1 above)
Comment (tramstheman) [inline Assignment_turn_in_module3.txt]: good catch.

File: Assignment_turn_in_module3.txt (hunk 2 above)
Comment (tramstheman) [inline Assignment_turn_in_module3.txt]: good catch

File: Assignment_turn_in_module3.txt [hunk 4]
```diff
@@ -1096,82 +1097,70 @@ The A of CLUBS was pulled from the deck.
...
+The A of CLUBS was pulled from the deck.
+The T of DIAMONDS was pulled from the deck.
+The 2 of CLUBS was pulled from the deck.
+The J of HEARTS was pulled from the deck.
+The J of CLUBS was pulled from the deck.
+The A of HEARTS was pulled from the deck.
+The 3 of HEARTS was pulled from the deck.
+The 9 of DIAMONDS was pulled from the deck.
+The 6 of HEARTS was pulled from the deck.
+The Cannot initialize deck. Number of cards either exceeds or does not reach the card boundaries.
```
Comment (GEvanscsumb) [inline Assignment_turn_in_module3.txt]: not sure if this was just a copy-paste error, otherwise everything else looks great, nice job!

Comment (tramstheman) [inline Assignment_turn_in_module3.txt]: not a copy-pasta error. This is what the output produced. I am guessing this is not intended?

**Comment by tramstheman** [comment]
Your fix is merged into this PR

**Pull request merged**